x-pack/plugins/watcher/common/constants/action_states.ts

```diff
@@ -5,41 +5,31 @@
  * 2.0.
  */
 
-import { i18n } from '@kbn/i18n';
-
+// These values are returned by the API and depended upon in both server-side
+// and client-side logic. TS will use these values to determine correctness.
```

Comment:
What do you think about using snakecase for the values  (`ok`, `config_error`, `acknowledged`) since these will not surface in the UI?

Reply:
Good idea. I'll implement this if I can't revert these changes, per my comment above.